charts/nginx-ingress-services/values.yaml

```diff
@@ -142,9 +142,8 @@ config:
 #
 # For Services:
 # service:
-#   s3:
-#     externallyCreated: true
-#     ^ externallyCreated might be useful if S3 access is provided by
+#   useFakeS3: true
+#     ^ useFakeS3 might be useful if S3 access is provided by
```

Comment:
```suggestion
#     ^ useFakeS3 should be enabled if S3 access is to be provided by fake-aws-s3, inside of the kubernetes cluster.
```

Reply:
this paragraph needs reworked. tl;dr: fakeAwsS3 should be used when s3 is to be provided by fake-aws-s3 within the cluster. when it is something outside of the cluster (like minio-external), we should leave this setting off. this setting creates a fake-aws-s3 service inside of the cluster, which should be what is referred to in the brig configuration.